tests/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py35,py36,py37
+skip_missing_interpreters = true
+skipsdist = true
+
+[testenv]
+deps=
+  -rrequirements.txt
+  pytest
+commands=
+    pytest -v {posargs:.}
```

Comment:
I checked [the docs](https://tox.readthedocs.io/en/latest/example/general.html) but wasn't able to find anything on it, would you elaborate what the `.` is doing in `{posargs:.}`?

Reply:
It's just running pytest in the current directory, totally not necessary I suppose. I find ``{posargs}`` useful though, if by some reason we ever need to point at a different test directory interactively.

Would you prefer if I remove that default value there? If it improves readability I'm ok with that.

Reply:
Ah I see now, that makes sense. For some reason I was thinking it was something for tox but you're just passing `.` through to pytest. That makes sense as the default functionality to me.

Reply:
This is also useful if one needs to pass any other pytest option to pytest run executed via tox, eg. like `tox -e py36 -- --pdb --runxfail tests/test_foo.py`.

That said, I don't see any value in using tox here to run ocsqe test cases.